plugins/netctl/cmdimpl/nodes.go

```diff
@@ -35,45 +31,17 @@ import (
 
 //PrintNodes will print out all of the cmdimpl in a network in a table format.
 func PrintNodes() {
-	cfg := &etcd.ClientConfig{
-		Config: &clientv3.Config{
-			Endpoints: []string{etcdLocation},
-		},
-		OpTimeout: 1 * time.Second,
+	nodes := make([]string, 0)
+	for k := range getClusterNodeInfo() {
```

Comment:
Seems like this part of sorting by node name is duplicated in PrintAllIpams() as well, perhaps it should be extracted into some function like: `getSortedNodeInfo() []*node.NodeInfo`, that returns all nodes sorted by name.

Reply:
That's why I proposed new function `getSortedNodeInfo()` to be added next to the old `getClusterNodeInfo()` and use it only in those cases where it's needed, simply to avoid these two cases, in `PrintNodes()` and `PrintAllIpams()`, of repeated sorting code and have both of them shortened to:

```go
for _, n := range getSortedNodeInfo() {
    // ...
}
```

Reply:
Merging to have this included in the release. Jan, please open another MR if you want to apply Ondrej's proposal.